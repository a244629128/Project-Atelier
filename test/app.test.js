import React from 'react';
import renderer from 'react-test-renderer';
import App from '../client/src/app';

test('App should display', () => {
  const component = renderer.create(
    <App>
    </App>,
  );

  let app = component.toJSON();
  expect(app).toMatchSnapshot();
});