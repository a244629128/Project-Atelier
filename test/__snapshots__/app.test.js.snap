// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should display 1`] = `
<div>
  <div>
    <div
      className="qna-component-name"
    >
      Questions and Answers
    </div>
    <div>
      Seach quastions
    </div>
    <div>
      Questions list
    </div>
    <div>
      More answered questions
    </div>
    <div>
      Add a new question
    </div>
  </div>
</div>
`;
